src/ejabberd_local.erl

```diff
@@ -114,7 +114,8 @@ process_iq(#iq{ xmlns = XMLNS } = IQ, Acc, From, To, _El) ->
         [{_, IQHandler}] ->
             gen_iq_component:handle(IQHandler, Acc, From, To, IQ);
         [] ->
-            ejabberd_router:route_error_reply(To, From, Acc, mongoose_xmpp_errors:feature_not_implemented())
+            ejabberd_router:route_error_reply(To, From, Acc,
+                mongoose_xmpp_errors:feature_not_implemented(<<"en">>, <<"From local handler">>))
```

Comment:
This error message, could be something like... `Local server does not implement this feature`, where the keyboard is the word `Local`. Should be a string that means something to the end user, and that points us to the place in code that generated it, at the same time.

Reply:
But again, maybe all these extended error messages could also be moved to a separate PR, this is a different topic to mam.